setup.py.in

```diff
@@ -10,14 +10,18 @@ with open("@CMAKE_SOURCE_DIR@/requirements.txt", "r") as fh:
 
 setup(
     name = "pairinteraction",
+    python_requires=">=3.8",
     version = r"@VERSION_WITHOUT_LEADING_V_COMPACT_ALPHA@",
     author="Sebastian Weber, Henri Menke, Johannes Block, Alexander Papageorge",
     description="A Rydberg Interaction Calculator",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://pairinteraction.github.io/",
     classifiers=[
-        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
```

Comment:
Is it really necessary to list all of them? Then we need to update whenever there is a new Python version and 3.12 which is currently in prerelease is already missing.

Reply:
To my understanding, these classifiers are necessary to have the different python versions listed in the "Programming Language" section on pypi. It seems to be more common to list there the specific Python versions instead of only listing `Python :: 3`. However, I do not have a strong opinion about it.

Reply:
Maybe then we only put 3.8 and consider it implied that any later version is also supported? If this is only for labeling then this should not pose a problem.

Reply:
Hmm, I think this can be confusing. I would either add `Programming Language :: Python :: 3` or all specific versions.